_userguides/Box_Panel.md

```diff
@@ -11,9 +11,9 @@ dateAdded: April 14, 2016
 
 ## Getting Started with Box Panel
 
-The Box Panel interface gives you a single, consolidated view of assets deployed in your **IBM Blue Box Dedicated** and **IBM Blue Box Local** cloud environments. Its self-service access lets you create support tickets, leverage central authentication for logins and user permissions, monitor various functions, and view your invoices, easily. 
+The Box Panel interface gives you a single, consolidated view of assets deployed in your **IBM Bluemix Dedicated on OpenStack** and **IBM Bluemix Local on OpenStack** cloud environments. Its self-service access lets you create support tickets, leverage central authentication for logins and user permissions, monitor various functions, and view your invoices, easily. 
 
-This user guide gives you an end-to-end overview, so you can get started using Box Panel to monitor and control your IBM Blue Box Cloud. This document applies to Box Panel Version 3.8.3 and later.
+This user guide gives you an end-to-end overview, so you can get started using Box Panel to monitor and control your IBM Bluemix on OpenStack. This document applies to Box Panel Version 3.8.3 and later.
```

Comment:
please add the word "environment"....should read "IBM Bluemix on OpenStack environment."